Gruntfile.js

```diff
@@ -18,21 +18,18 @@ module.exports = function(grunt) {
                     exclusionPattern: /browser/
```

Comment:
It's generally a bad practice to make whitespace-only changes to a file in a PR.